README.md

```diff
@@ -73,8 +73,7 @@ To install from source after cloning this repository, simply run
 pip install -e .
 ```
 
-Development
-+++++++++++
+### Development
```

Comment:
should't this heading be at the same level as the one above (Install)?

Reply:
If you wish, my thought was that it's install instructions for development, rather than just install (which is the heading above)

Reply:
@gidden, what's your take here?